Homework_1.py

```diff
@@ -0,0 +1,43 @@
+# importing libraries
+
+import pandas as pd
+import plotly.express as px
+from sklearn.preprocessing import StandardScaler
+
+data = pd.read_csv("/home/osboxes/Downloads/iris.csv")
```

Comment:
This is hardcoded path specific to your system environment. It will not work on other PCs

Reply:
Resolved this issue by importing dataset from sklearn.